developer/src/kmc-kmn/src/kmw-compiler/validate-layout-file.ts

```diff
@@ -239,7 +242,8 @@ export function ValidateLayoutFile(fk: KMX.KEYBOARD, FDebug: boolean, sLayoutFil
 
   let hasWarnedOfGestureUseDownlevel = false;
   const warnGesturesIfNeeded = function(keyId: string) {
-    if(!hasWarnedOfGestureUseDownlevel && !IsKeyboardVersion17OrLater()) {
+    // TODO: automatic version upgrade
+    if(!hasWarnedOfGestureUseDownlevel && !isKeyboardVersion17OrLater()) {
```

Comment:
TODO: automatically upgrade version when gestures are used in KeymanWeb keyboard